src/schematools/types.py

```diff
@@ -660,24 +688,35 @@ def __init__(
         self._required = _required
         self._temporal = _temporal
 
+    @classmethod
+    def from_dict(cls, obj: Dict[str, Any]) -> DatasetFieldSchema:
+        """Create fieldschema from a dict.
+
+        When this factory function is used, the _id and _parent_table arguments are missing
+        and will be set to the "id" of the obj argument and an empty DatasetSchema.
+        """
+        return cls(
+            obj["id"], DatasetTableSchema(DatasetSchema({"id": "parent", "type": "dataset"})), obj
```

Comment:
is dat handig, dat er een fake tabel in gezet wordt? Geeft ook weer andere rare fouten

Reply:
Verwijderd.